index.html

```diff
@@ -17,6 +17,11 @@
 
     <title>My app</title>
     <meta name="description" content="My App description">
+    
+    <!-- Performance tip: hint to the browser to fetch and cache webcomponents-loader.js -->
+    <link rel="preload" as="script" href="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js">
```

Comment:
hmm not sure about this one. my gut feeling says that out of the browsers that preload works on, chrome/safari don't need the polyfill (and therefore the loader), only FF and Edge do (and soon: only Edge). IE which needs the most polyfills (and benefits from this being downloaded earlier) doesn't have preload, and won't benefit from this. It sounds like moving this earlier will hose performance where the loader isn't needed at all. Do you have any WebPageTest or Lighthouse comparisons that show this actually helps?

Reply:
I just did a few tests with lighthouse for both desktop with no throttling and mobile with throttling and the results were almost identical with and without the preload hint so you are totally right it's probably not worth making the change.

Reply:
What are your thoughts on moving the preconnect hint for the fonts higher up in the header? :)

Reply:
I highly doubt moving the link will make any difference in performance at all. I'd suggest to just fix the typo in "Performace"

Reply:
@frankiefu ok I will change the pull request to only correct the typo